docs/sabakan-integration.md

```diff
@@ -328,6 +329,13 @@ Following Machine fields are translated to Node annotations:
 | `spec.registerDate` | `cke.cybozu.com/register-date` | `spec.registerDate` in RFC3339 format. |
 | `spec.retireDate`   | `cke.cybozu.com/retire-date`   | `spec.retireDate` in RFC3339 format.   |
 
+Dynamic Node Labels and Annotations
+-----------------------------------
+
+CKE can also create labels in combination with fields under Sabakan [`Machine`][schema] `Spec`.
+Only values of the labels are created with go-template (e.g. `rack: {{ .Rack }}` ).
+The available fields can be referenced in [Labels](#labels) and [Annotations](#annotations).
```

Comment:
Why do you link to "static" labels and annotations?

Please add some examples for dynamic labels and annotations instead of referencing static ones.